### PR TITLE
Changed spelling of whitch to which

### PR DESCRIPTION
**Pull request by iansmellis**

### CODE REVIEW

File: Sources/SpaceEngineers.Game/SpaceEngineersGame.cs [hunk 1]
```diff
@@ -47,7 +47,7 @@ public static void SetupPerGameSettings()
                 {
                     Mode = MyGridPlacementSettings.SnapMode.OneFreeAxis,
                     SearchHalfExtentsDeltaRatio = -0.1f,
-                    SearchHalfExtentsDeltaAbsolute = -0.13f, //this is value at whitch you can place new small ship and is ok for wheels too
+                    SearchHalfExtentsDeltaAbsolute = -0.13f, //this is value at which you can place new small ship and is ok for wheels too
```
Comment (harag-on-steam) [inline Sources/SpaceEngineers.Game/SpaceEngineersGame.cs]: Spelling in a comment as the only change?

**Comment by IhanaMies** [comment]
One word, why? This PR server no purpose what so ever.

**Comment by Spartan322** [comment]
2 commits and a merge only for a single correction...in a comment? Close this.

**Comment by iansmellis** [comment]
every little bit helps right?

**Comment by mccorkle** [comment]
It actually cost Keen engineering time to merge in a PR,  so if there is no
immediate value to your PR,  I'm guessing they won't merge it, but it will
leave another open PR for them (and any other contributor) to have to check
before they begin work on something so as to not duplicate work.  I believe
this is why the request came to close this.

If you really want to fix up the spelling and other grammar in comments,
bring those with solid code changed to help the code base improve.  Or help
us all with more documentation.

**Comment by iansmellis** [comment]
Thank you for a much better explanation mccorkle, I will close my request now.

**Pull request closed without merging**